src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -8,9 +8,10 @@
     <InboxOnTargetFramework Include="net45">
       <AsFrameworkReference>true</AsFrameworkReference>
     </InboxOnTargetFramework>
-    <PackageFile Include=".\build\System.Speech.targets">
-      <TargetPath>build\netcoreapp2.0\</TargetPath>
-    </PackageFile>
+    <PackageFile Include="buildTransitive\System.Speech.targets"
```

Comment:
Consider removing the condition that's in this targets file.

Reply:
Funny that I didn't think about this. Thanks :)